docker-compose/camunda-8.6/docker-compose.yaml

```diff
@@ -353,7 +374,8 @@ services:
 
   modeler-db:
     profiles:
-      - web-modeler
+      - full
+      - web-modeler-only
```

Comment:
I wasn't able to rename `modeler-db` to `web-modeler-db` because I ran into an authentication error which I couldn't resolve. I could not fix it in a timely manner.

Reply:
I added https://github.com/camunda/camunda-platform/pull/657/commits/d789e236123ebbe914f13947887e7919c1b9df6f to implement the rename to `web-modeler-db`.